packages/components/src/components/data-list/data-list-row.jsx

```diff
@@ -4,8 +4,18 @@ import { NavLink } from 'react-router-dom';
 
 const DataListRow = props => {
     const [show_desc, setShowDesc] = React.useState(false);
-    const { destination_link, acion_desc, row_key, row_gap } = props;
-    const { rowRenderer, row, measure, is_scrolling, is_new_row } = props;
+    const {
+        acion_desc,
+        destination_link,
+        is_new_row,
+        is_scrolling,
+        is_top_up,
```

Comment:
`DataList` is a generic component. We should avoid destructuring logic specific props here. Maybe we can destructure with the rest operator `(...other_props)` and pass it down to the children.

Reply:
Sure, that's a good suggestion 👍🏻